src/content/self-hosted/install/backup-restore.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup and Restore
+description: Backup and Restore your Okteto instance
+id: backup
+sidebar_label: Backup and Restore
+---
+
+Backup and restore planning is essential to ensure the availability and integrity of your Okteto instance in case of unexpected failures or disaster recovery.
+This guide outlines the steps to recover an Okteto instance in such scenarios.
+
+## Critical Resources
+
+To effectively recover an Okteto instance, focus on the following critical resources:
+
+- **Service Accounts**: Preserve service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
```

Comment:
```suggestion
- **Service Accounts**: Preserve any service accounts associated with users. These service accounts reside within the `okteto` namespace and are labeled with `dev.okteto.com: "true"`.
```

Small suggested change. 

Do we want to explain _why_ these service accounts are so important for recovery? For example: does this save a lot of configuration/setup time? I think including rationale to explain why this specific resource is important may help here

Reply:
@omnomagonz makes sense, I will add an explanation